irods/manager/data_object_manager.py

```diff
@@ -47,7 +47,7 @@ def _download(self, obj, local_path, auto_parallel = AUTO_SWITCH_PARALLEL, **opt
             parallel_xfer = False
             if auto_parallel and self.server_version >= [4,2,8]:
                 dobj_size = o.seek(0, os.SEEK_END)
-                if dobj_size > PARALLEL_THRESHOLD:
+                if dobj_size >= PARALLEL_THRESHOLD:
```

Comment:
icommands have it the other way.   max for single buffer = 32MB...

https://github.com/irods/irods/blob/4-2-stable/server/api/src/rsDataObjOpen.cpp#L505

so remove the '=' to be consistent